indico/modules/categories/controllers/management.py

```diff
@@ -102,6 +102,12 @@ def _process(self):
             flash(_("Category settings saved!"), 'success')
             return redirect(url_for('.manage_settings', self.category))
         else:
+            # Check if category visibility would be affected by any of the parents
```

Comment:
Wouldn't it make more sense to have this code in the form's constructor? You pass the category to it anyway